spec/lib/hackney/income/create_agreement_spec.rb

```diff
@@ -93,7 +93,8 @@
       breached_agreement = Hackney::Income::Models::Agreement.create(
         tenancy_ref: tenancy_ref,
         current_state: 'breached',
-        created_by: created_by
+        created_by: created_by,
+        agreement_type: 'informal'
```

Comment:
tiny tiny nit pick 
```suggestion
        agreement_type: :informal
```
Symbols are cool, string are not